pandas/io/json/_json.py

```diff
@@ -655,11 +666,16 @@ def __init__(
         self.chunksize = chunksize
         self.nrows_seen = 0
         self.should_close = False
+        self.nrows = nrows
 
         if self.chunksize is not None:
             self.chunksize = _validate_integer("chunksize", self.chunksize, 1)
             if not self.lines:
                 raise ValueError("chunksize can only be passed if lines=True")
+        if self.nrows is not None:
+            self.chunksize = _validate_integer("nrows", self.nrows, 0)
```

Comment:
```suggestion
            self.nrows = _validate_integer("nrows", self.nrows, 0)
```

I think this is a typo

Reply:
Thanks @WillAyd

Reply:
Done, you may review again